tensorflow/lite/tools/benchmark/README.md

```diff
@@ -87,6 +87,13 @@ and the following optional parameters:
     `enable_op_profiling`. When this is set to true the profile of ops
     on hexagon DSP will be added to the profile table.
     Note that, the reported data on hexagon is in cycles, not in ms like on cpu.
+*   `use_external_delegate`: `bool` (default=false) \
```

Comment:
Should we remove this flag and just make it implicit based on `external_delegate_path`?

Reply:
I think it makes sense to make it implicit, will remove.